subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -475,6 +475,18 @@ config BT_CTLR_CONN_RSSI_EVENT
 	help
 	  Generate events for connection RSSI measurement.
 
+config BT_CTLR_ALLOW_SAME_PEER_CONN
+	bool "Allow connection requests from same peer"
+	depends on BT_MAX_CONN > 1
+	help
+	  Allow connection requests from the same peer. While the
+	  Bluetooth specification does not allow multiple connections
+	  with the same peer, allowing such connections is useful
+	  while debugging multiple connections.
+
+	  WARNING: This option enables behavior that violates the Bluetooth
+	  specification.
```

Comment:
I would like @carlescufi input on whether this is enough, or if we also should add build warnings like was done for other such configurations in https://github.com/zephyrproject-rtos/zephyr/pull/26186

Reply:
> I would like @carlescufi input on whether this is enough, or if we also should add build warnings like was done for other such configurations in #26186

@carlescufi What is your take in this?

Reply:
@wopu-ot Is this something that is critical to get in? I can mark it as approved, and then we can consider adding additional warnings (if needed) afterwards in a separate PR.